Example/ConsentViewController_ExampleTests/SPClientCoordinator/SPClientCoordinatorSpec.swift

```diff
@@ -42,8 +42,45 @@ class SPClientCoordinatorSpec: QuickSpec {
             )
         }
 
+        describe("selectPrivacyManagerId") {
```

Comment:
@Nevazhnovu I believe this test doesn't belong to `SPClientCoordinatorSpec`, but rather `SPConsentManagerSpec` (feel free to create the file if that doesn't exist yet).